src/geos.h

```diff
@@ -13,13 +13,92 @@
 #pragma GCC diagnostic pop
 
 
+/* Macros to setup GEOS Context and error handlers 
+
+Typical PyGEOS pattern in a function that uses GEOS:
+
+GEOS_INIT;   // or GEOS_INIT_THREADS if you use no CPython calls
```

Comment:
Suggestion, it might be useful to note that `GEOS_INIT` makes `ctx` available, and that `ctx` is only valid between `GEOS_INIT` and `GEOS_FINISH`.